src/components/Menu/LanguageNavigation/LanguageNavigation.tsx

```diff
@@ -85,30 +110,50 @@ const LanguageNavigation = ({ items, localChangeOnly, selectedLanguage, onSelect
   const value = options.find((option) => option.value === actualSelectedLanguage);
 
   const shouldUseLocalChanges = localChangeOnly && !!onSelect;
-  const onSelectChange = shouldUseLocalChanges ? onSelect : changePageOnSelect(pageLanguage);
+  const onSelectChange = shouldUseLocalChanges
+    ? onSelect
+    : changePageOnSelect(pageLanguage, DEFAULT_PREFERRED_INTERFACE);
+  const selectedSDK = getSDKInterface();
+  const realtimeCode = getLanguageItemsIfHasSDKInterface(items, 'rt');
+  const restCode = getLanguageItemsIfHasSDKInterface(items, 'rest');
 
-  return items.length >= 2 ? (
+  return (
     <>
-      <SDKNavigation
-        items={items}
-        localChangeOnly={localChangeOnly}
-        selectedLanguage={selectedLanguage}
-        onSelect={onSelect}
-      />
-      <div className="border-b border-charcoal-grey w-full">
-        <menu data-testid="menu" className={horizontalNav}>
-          {items.map(({ Component, props, content }, index) => (
-            <Component {...props} key={index}>
-              {content}
-            </Component>
-          ))}
-          <div className={dropdownContainer}>
-            <Select options={options} value={value} isSearchable={false} onChange={onSelectChange} />
-          </div>
-        </menu>
-      </div>
+      {realtimeCode.includes(true) || restCode.includes(true) ? (
+        <SDKNavigation
+          items={items}
+          localChangeOnly={localChangeOnly}
+          selectedLanguage={selectedLanguage}
+          onSelect={onSelect}
+          SDKSelected={selectedSDK}
+        />
+      ) : null}
+
+      {items.length >= 2 ? (
+        <div className="border-b border-charcoal-grey w-full">
+          <menu data-testid="menu" className={horizontalNav}>
+            {items.map(({ Component, props, content }, index) => (
+              <Component {...props} key={index}>
+                {content}
+              </Component>
+            ))}
+            <div className={dropdownContainer}>
+              <Select options={options} value={value} isSearchable={false} onChange={onSelectChange} />
+            </div>
+          </menu>
+        </div>
+      ) : null}
     </>
-  ) : null;
+  );
 };
 
 export default LanguageNavigation;
+
+const getLanguageItemsIfHasSDKInterface = (
+  items: {
+    Component: FC<LanguageNavigationComponentProps>;
+    props: LanguageNavigationComponentProps;
+    content: string;
+  }[],
+  sdkInterface: string,
+) => items.map((languageItems) => languageItems.props.language.includes(sdkInterface));
```

Comment:
Just a small change here, I'd suggest we make it `languageItem`

Reply:
@ferdipret  thanks for spotting this. Its now updated